clusterDeployer.py

```diff
@@ -103,6 +110,58 @@ def setup_all_vms(h: host.Host, vms, iso_path, virsh_pool) -> list:
     return futures
 
 
+def configure_bridge(h) -> None:
+    cmd = "systemctl enable libvirtd"
+    run_cmd(h, cmd)
+    cmd = "systemctl start libvirtd"
+    run_cmd(h, cmd)
+
+    # stp must be disabled or it might conflict with default configuration of some physical switches
+    # 'bridge' section of network 'default' can't be updated => destroy and recreate
+    # check that default exists and contains stp=off
+    cmd = "virsh net-dumpxml default"
+    ret = h.run(cmd)
+
+    if "stp='off'" not in ret.out:
+        logger.info("=================== !!!!!!!!!!!!!!!!!!!!!! ====================")
+        logger.info(" !!!!!!!!!!!!! Destoying and recreating bridge !!!!!!!!!!!!!!!!")
+        logger.info("=================== !!!!!!!!!!!!!!!!!!!!!! ====================")
```

Comment:
I'm not a fan of this way to log things :). Can we do more subtle log? with grep you will still see it.
Also, it seems like this is better suited for `logger.debug`